deno/payloads/v10/channel.ts

```diff
@@ -1916,6 +1916,8 @@ export interface APIMessageSnapshot {
 	message: APIMessageSnapshotFields;
 	/**
 	 * Id of the origin message's guild
+	 *
+	 * @deprecated This field doesn't accurately reflect the Discord API as it doesn't exist nor is documented.
```

Comment:
Please add in what Jira said + that it will be removed in the next major version